registry.json

```diff
@@ -5489,6 +5489,19 @@
    "telegram": "holomorphix",
    "operatorName": "Ryan & Jorden",
    "github": "holomorphix"
-
+  },
+  {
+  "ticker": "WAX",
+   "poolId": "pool1vurlp3ndgge8avz3h8lu2wzdqcwc8w4l5s43rzlk8p4pz0x5vla",
```

Comment:
Pool Id is wrong, in HEX format WAX pool is :

6707f0c66d42327eb051b9ffc5384d061d83babfa42b118bf6386a11